tests/testthat/test-db-interface.R

```diff
@@ -197,3 +197,60 @@ test_that("copy_dm_to() fails with schema argument for databases other than MSSQ
     "no_schemas_supported"
   )
 })
+
+
+test_that("build_copy_queries snapshot test for pixarfilms", {
+  src_db <- my_db_test_src()
+
+  # build regular dm from `dm_pixarfilms()`
+  pixar_dm <-
+    # fetch sample dm
+    dm_pixarfilms() %>%
+    # make it regular
+    dm_filter(pixar_films, !is.na(film)) %>%
+    dm_apply_filters() %>%
+    dm_select_tbl(-pixar_people)
+
+  skip_if_not_installed("testthat", "3.1.0")
+
+  expect_snapshot(
+    variant = if (packageVersion("testthat") > "3.1.0") "testthat-new" else "testthat-legacy",
+    pixar_dm %>%
+      build_copy_queries(
+        src_db,
+        .,
+        table_names = names(.) %>%
+          repair_table_names_for_db(temporary = FALSE, con = src_db, schema = NULL) %>%
+          map(dbplyr::ident_q)) %>%
+      map(as.data.frame) # to print full queries
+  )
```

Comment:
Braces are important for srcrefs. I'd like the query to be printed on several lines, we could use `glue::as_glue()` for the `sql` component.

```suggestion
  expect_snapshot({
    variant = if (packageVersion("testthat") > "3.1.0") "testthat-new" else "testthat-legacy",
    pixar_dm %>%
      build_copy_queries(
        src_db,
        .,
        table_names = names(.) %>%
          repair_table_names_for_db(temporary = FALSE, con = src_db, schema = NULL) %>%
          map(dbplyr::ident_q)) %>%
      map(as.list) # to print full queries
  })
```

Reply:
With `as.list()` instead of `as.data.frame` queries are printed on several lines already because they have the SQL class and printing method.